packages/manager/modules/pci/src/projects/project/storages/databases/add/add.utils.js

```diff
@@ -0,0 +1,110 @@
+export function getOrderDataFromModel(model) {
+  const orderData = {
+    description: model.name,
+    nodesPattern: {
+      flavor: model.flavor.name,
+      number: model.plan.nodesCount,
+      region: model.region.name,
+    },
+    plan: model.plan.name,
+    version: model.engine.selectedVersion.version,
+    disk: {
+      size: model.disk.additionalDiskSize
+        ? model.disk.initialSize + model.disk.additionalDiskSize
+        : model.disk.initialSize,
+    },
+  };
+  if (model.usePrivateNetwork && model.subnet?.id?.length > 0) {
+    orderData.networkId = model.privateNetwork.regions?.find(
+      (region) => region.region === model.subnet.ipPools[0]?.region,
+    )?.openstackId;
+    orderData.subnetId = model.subnet.id;
+  }
+  return orderData;
+}
+
+export function getTerraformDataFromModel(projectId, model) {
+  const terraformData = {
+    key: 'resource',
+    values: ['ovh_cloud_project_database', model.engine.name],
+    nodes: [
+      {
+        key: 'service_name',
+        values: [projectId],
+        nodes: [],
+      },
+      {
+        key: 'description',
+        values: [model.name],
+        nodes: [],
+      },
+      {
+        key: 'engine',
+        values: [model.engine.name],
+        nodes: [],
+      },
+      {
+        key: 'version',
+        values: [model.engine.selectedVersion.version],
+        nodes: [],
+      },
+      {
+        key: 'plan',
+        values: [model.plan.name],
+        nodes: [],
+      },
+
+      {
+        key: 'flavor',
+        values: [model.flavor.name],
+        nodes: [],
+      },
+      {
+        key: 'disk_size',
+        values: [
+          model.disk.additionalDiskSize
+            ? model.disk.initialSize + model.disk.additionalDiskSize
+            : model.disk.initialSize,
+        ],
```

Comment:
The format differs between `API` and `Terraform`:
```
disk: {
    size: 100
}
```
```
"disk_size": 100
```
Is it intended ? I expected something like the nodes for `Terraform`:
```
"disk" {
    "size" = 100
}
```

Reply:
I had checked with the product Team and we will use for `API`: 

```json
disk: {
    size: 100
}
```

and for `Terraform` this naming `disk_size`.